assets/js/theme/common/carousel/utils/dotsSetup.js

```diff
@@ -0,0 +1,21 @@
+export default ($dots, currentSlide, realSlideCount, dotLabels) => {
```

Comment:
actualSlideCount ?

Reply:
For example we have 10 product items. On the first glance we have 10 slides, but we know that on one slide we should have 6 items  slide. Result - 2 slides. So 10 items, but real 2 slides for them. So if you do not insist - real in this context is closer for me:)

Reply:
Agree with Alex, we should use actual instead of real

Reply:
if you both think actual is better - no problem